media-gfx/openvdb/openvdb-3.2.0.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $id$
+
+EAPI="6"
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit eutils python-r1
+
+DESCRIPTION="Libs for the efficient manipulation of volumetric data"
+HOMEPAGE="http://www.openvdb.org"
+
+SRC_URI="https://github.com/dreamworksanimation/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+openvdb-compression doc pdfdoc python X"
```

Comment:
Too much choice. `USE=pdfdoc` is not Gentoo-idiomatic, and no user likely expects setting such a flag. Unify the semantics of `USE=pdfdoc` and `USE=doc` to be covered by just setting `USE=doc`.